src/runtime/pkg/oci/utils.go

```diff
@@ -485,6 +485,12 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig,
 		return err
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.ImagePath]; ok {
```

Comment:
Do we need to consider a vc.HypervisorType check here too. I'm conscious that the ImagePath annotation was/is handled in `addAssetAnnotations`, so am not sure if there is any negative effects of setting it here too.

Reply:
Alternatively could we remove this code completely and handle it in sandbox.go?